test/integration/members.test.js

```diff
@@ -135,25 +129,28 @@ describe("Members", function () {
   });
 
   describe("GET /members/idle", function () {
-    before(async function () {
-      await cleanDb();
+    beforeEach(async function () {
+      await addUser();
     });
+
     it("Should return empty array if no idle member is found", function (done) {
-      chai
-        .request(app)
-        .get("/members/idle")
-        .end((err, res) => {
-          if (err) {
-            return done(err);
-          }
+      cleanDb().then(() => {
```

Comment:
Is after each doesn't taking care of this ??

Reply:
After each cleans the DB but then beforeEach adds the users in the DB.

Reply:
I'm saying you have already added cleanDb in afterEach, so calling this function here and doing something on resolving the promise, is this necessary to do so ??

Reply:
As discussed we'll we creating separate describe blocks for the 2 cases : when there are no users in the DB and when there are users in DB